src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,7 @@
 package core.basesyntax;
 
 public class MainApp {
-
+public static void main(String[] args) {
+    System.out.println("First solutiong");
```

Comment:
The message printed to the console should be informative and indicate what type of `Machine` is working. The word 'solutiong' seems to be a typo and is not informative. Consider revising the message to reflect the operation of a specific machine, such as 'Truck', 'Bulldozer', or 'Excavator'.